kinto/plugins/history/views.py

```diff
@@ -57,3 +61,85 @@ def _extract_filters(self):
             filters_str_id.append(filt)
 
         return filters_str_id
+
+
+version_view = Service(name='version_view',
+                       description='Handle retrieving object from the past.',
+                       path='{subpath:.*}/version/{last_modified:[0-9]{13}}')
+
+
+@version_view.get(permission=NO_PERMISSION_REQUIRED)
```

Comment:
Does this make every snapshot public?

The permission aspect is crucial.
Example scenario: I have access to a collection, then I am removed the perm, then some data is created. Can I access a snapshot of some data that was edited after I was removed the permission?

It's not impossible to handle, but not trivial.

The history endpoint already handles this, by providing history entries only on data that is readable/writable. That's why I suggested to do this reconciliation on the client side.